src/test/java/com/microsoft/sqlserver/jdbc/bulkCopy/BulkCopyTimestampTest.java

```diff
@@ -0,0 +1,111 @@
+package com.microsoft.sqlserver.jdbc.bulkCopy;
+
+import com.microsoft.sqlserver.jdbc.*;
+import com.microsoft.sqlserver.testframework.AbstractSQLGenerator;
+import com.microsoft.sqlserver.testframework.AbstractTest;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.platform.runner.JUnitPlatform;
+import org.junit.runner.RunWith;
+
+import javax.sql.RowSetMetaData;
+import javax.sql.rowset.CachedRowSet;
+import javax.sql.rowset.RowSetFactory;
+import javax.sql.rowset.RowSetMetaDataImpl;
+import javax.sql.rowset.RowSetProvider;
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+
+@RunWith(JUnitPlatform.class)
+public class BulkCopyTimestampTest extends AbstractTest {
+
+    public static final int COLUMN_COUNT = 16;
+    public static final int ROW_COUNT = 10000;

```

Comment:
it's not really necessary to do this that many times, just make sure this is run at last once cover the newly added code